### PR TITLE
travis ci

### PR DESCRIPTION
**Pull request by danethurber**

### CODE REVIEW

File: packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports[`Storyshots as link hrefs 1`] = `
     data-css-nipl9k=""
   >
     <a
-      data-css-1o2hsrq=""
+      data-css-1zmsb0=""
```
Comment (danethurber) [inline packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap]: the <Button /> css changed so all these related snapshots needed to change

Comment (jaketrent) [reply packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap]: when/why did the button css change?

File: dangerfile.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { danger, warn } from 'danger'
```
Comment (danethurber) [inline dangerfile.js]: this is a staring place for automating helpful things do make prs easier to manage. thoughts? suggestions?

**Comment by danethurber** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  node` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh
Error:  Error: Plugin/Preset files are not allowed to export objects, only functions. In /home/travis/build/pluralsight/design-system/node_modules/babel-preset-react/lib/index.js
    at createDescriptor (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:178:11)
    at items.map (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPresetDescriptors (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
    at presets (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:47:19)
    at mergeChainOpts (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-chain.js:320:26)
    at /home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-chain.js:283:7
    at buildRootChain (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-chain.js:120:22)
    at loadPrivatePartialConfig (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/partial.js:85:55)
danger-results://tmp/danger-results.json
```

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a> against d75b0faf62692072b4ad76669f8fa29e72179e4e
</p>

File: packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap (hunk 1 above)
Comment (danethurber) [reply packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap]: here i believe -> https://github.com/pluralsight/design-system/commit/df31e9a2dde46417a897b9455847075c9b379f00

**Comment by danethurber** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  node` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh
Error:  Error: Plugin/Preset files are not allowed to export objects, only functions. In /home/travis/build/pluralsight/design-system/node_modules/babel-preset-react/lib/index.js
    at createDescriptor (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:178:11)
    at items.map (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPresetDescriptors (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
    at presets (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-descriptors.js:47:19)
    at mergeChainOpts (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-chain.js:320:26)
    at /home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-chain.js:283:7
    at buildRootChain (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/config-chain.js:120:22)
    at loadPrivatePartialConfig (/home/travis/build/pluralsight/design-system/node_modules/@babel/core/lib/config/partial.js:85:55)
danger-results://tmp/danger-results.json
```

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a> against d75b0faf62692072b4ad76669f8fa29e72179e4e
</p>

File: packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap (hunk 1 above)
Comment (jaketrent) [reply packages/breadcrumb/src/react/__specs__/__snapshots__/storyshots.spec.js.snap]: I see, an unrelated previous change to button broken snapshots where the other component used button, and we just happen to fix that here.  makes sense, thank you.

**Comment by danethurber** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  node` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh
Error:  TypeError: Cannot read property 'code' of null
    at Object.exports.babelify (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/utils/transpiler.js:133:19)
    at Object.exports.default (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/utils/transpiler.js:152:26)
    at Object.<anonymous> (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/inline.js:132:48)
    at step (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/inline.js:32:23)
    at Object.next (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/inline.js:13:53)
    at /home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/inline.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/inline.js:3:12)
    at exports.runDangerfileEnvironment (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/runner/runners/inline.js:105:136)
    at Object.<anonymous> (/home/travis/build/pluralsight/design-system/node_modules/danger/distribution/platforms/GitHub.js:167:38)
danger-results://tmp/danger-results.json
```

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a> against f1da3658068349428e0b4e8e8c0e209781f40cf1
</p>

**Comment by danethurber** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :grey_question: This PR does not include a description.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>:exclamation: No Reviewer: Please add some teammates to review your PR.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a> against 9ade9b052cc9730f28b60659fa7dfab819d10e74
</p>

**Comment by danethurber** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :grey_question: This PR does not include a description.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>:exclamation: No Reviewer: Please add some teammates to review your PR.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a> against d1cee75f7402da83bd98d4fb76f0ebb5a6be9077
</p>

**Pull request merged**